packages/commonwealth/server/eventHandlers/notifications.ts

```diff
@@ -65,7 +66,7 @@ export default class extends IEventHandler {
       formattedEvent.ChainEvent.ChainEventType = dbEventType.toJSON()
 
       // publish notification to the appropriate RabbitMQ queue (sends to socket.io servers to send to clients)
-      await this._rabbitMqController.publish(formattedEvent, 'ChainEventsNotificationsPublication')
+      await this._rabbitMqController.publish(formattedEvent, RascalPublications.ChainEventNotifications)
```

Comment:
rather than the comment here, how can we make the function name more implicit. Perhaps literally, `publishToSocketIo` then the code itself is self explanatory. (I know you didn't write this comment but just a side thought)